docs/_extra/api-reference/schemas/new-group.yaml

```diff
@@ -0,0 +1,14 @@
+Group:
+  type: object
+  properties:
+    name:
+      type: string
+      description: The name of the new group
+      minLength: 3
+      maxLength: 250
+    description:
+      type: string
+      description: An optional description for the new group
```

Comment:
Probably not necessary to say "optional" here as the rendered docs will mark it as required or not